options/m_option.c

```diff
@@ -1792,10 +1796,11 @@ static int parse_color(struct mp_log *log, const m_option_t *opt,
 error:
     mp_err(log, "Option %.*s: invalid color: '%.*s'\n",
            BSTR_P(name), BSTR_P(param));
-    mp_err(log, "Valid colors must be in the form #RRGGBB or #AARRGGBB (in hex)\n"
-                "Or in the form 'r/g/b/a', where each component is a value in the\n"
-                "range 0.0-1.0. (Also allowed: 'gray', 'gray/a', 'r/g/b'.\n");
-    return M_OPT_INVALID;
+print_help:
```

Comment:
Would probably be better to have only one label (at the position of the error label), and print the error only if `!is_help`?